src/modules/home/screens/home-screen.tsx

```diff
@@ -1,45 +1,50 @@
-import { Text, View, Image, StyleSheet, ScrollView } from 'react-native';
-import { SafeAreaView } from 'react-native-safe-area-context';
-import CarouselComponent from './carousel-component';
 import { ScreenWidth } from '@rneui/base';
+import { Image, ScrollView, StyleSheet, Text, View } from 'react-native';
+
+import { common } from '../../../translations/en.json';
+import { SafeAreaView } from '../../../ui/core/view';
+import CarouselComponent from './carousel-component';
 import PaymentMethods from './payment-methods';
 
 export const HomeScreen = () => {
-
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView>
         <View>
           <Image
             style={styles.logo}
-            source={require('../../../ui/assets/images/logo-black-market/Header.png')}></Image>
+            source={require('../../../ui/assets/images/logo-black-market/Header.png')}
+          />
         </View>
-        <CarouselComponent></CarouselComponent>
+        <CarouselComponent />
         <View style={styles.promotionsContainer}>
           <Image
             source={require('../../../ui/assets/images/home-page/home-page-middle-image.png')}
             style={styles.promotionImage}
           />
           <View style={styles.promotionTextContainer}>
-            <Text style={{ color: '#FFFFFF', fontWeight: 'bold', width: '80%', marginBottom: 5}}>
-              Check out our new and restored furniture
+            <Text style={{ color: '#FFFFFF', fontWeight: 'bold', width: '80%', marginBottom: 5 }}>
+              {common.homeScreen.checkOutPromotion}
+            </Text>
+            <Text style={{ color: '#FFFFFF', width: '80%' }}>
```

Comment:
Please move your styles to a separate file. I recommend to activate the no-inline-style rule :)

Reply:
Done.